README.md

```diff
@@ -206,13 +206,19 @@ Once these flags are set, your Fenix builds will include any local modifications
 
 See a [demo of auto-publication workflow in action](https://www.youtube.com/watch?v=qZKlBzVvQGc).
 
+In order to build successfully, you need to check out a commit in the dependency repository that has no breaking changes. The two best ways to do this are:
+- Run the `<android-components>/tools/list_compatible_dependency_versions.py` script to output a compatible commit
```

Comment:
This is more of a question, but should we mention that in order to run it, you need to clone A-C? I'm asking since I'm not sure who our target users are. If we think some of our users are going to be first timers with A-C / GV it might be worthwhile to mention it imo.

Reply:
Good question – You need ac (or GV) to do local substitution so cloning ac is kind of a requirement. It seems maybe a little too explicit to add this here, I think but if you disagree, let me know.